nexus/go-handler.md

```diff
@@ -0,0 +1,258 @@
+# Temporal Go SDK - Nexus Handler
+
+## Overview
+
+An addition to the [Temporal Go SDK][temporal-go-sdk] that exposes the ability for users to handle Nexus API requests.
+
+**Goals**
+
+- Expose a simple, high level API for binding Nexus operations to common Temporal constructs
+- Support for starting and canceling operations
+- Support multi-tenant applications with minimal boilerplate
```

Comment:
should I read this as "support cross namespace calls?"

Reply:
That's part of it but we want to allow calling from different namespaces possibly from different organizations.